### PR TITLE
Add test for wanedge

### PR DESCRIPTION
**Pull request by akry**

### CODE REVIEW

**Comment by akry** [comment]
wanedge test is now quite unstable because translation_manager and translations/static_address are racy.

If it succeeds the flow table 35 in wanedge shows the following flows:
```
(35): TABLE_ROUTE_EGRESS_LOOKUP
 35-00        0       0 => SWITCH(0x0)               actions=drop
 35-10        0       0 => ACTIVE_INTERFACE(0xd)    metadata=VALUE_PAIR(0x8000000a00000000/0xffffffff00000000)(0x0) actions=goto_table:TABLE_LOOKUP_IF_RL_TO_DP_RL(71)
 35-20        9       0 => INTERFACE(0xa)[0x12]     metadata=VALUE_PAIR(0x8000000a00000000/0xffffffff00000000)(0x0) actions=write_metadata:0x70200000000000a,goto_table:TABLE_ROUTE_EGRESS_TRANSLATION(36)
 35-20        0       0 => INTERFACE(0xb)[0x12]     metadata=VALUE_PAIR(0x8000000b00000000/0xffffffff00000000)(0x0) actions=write_metadata:0x70200000000000b,goto_table:TABLE_ROUTE_EGRESS_TRANSLATION(36)
 35-50       10       0 => TRANSLATION(0x1)         ip,metadata=VALUE_PAIR(0x8000000b00000001/0xffffffff7fffffff)(0x1),nw_src=10.101.0.10 actions=set_field:10.210.0.104->ip_src,write_metadata:0x70200000000000b,goto_table:TABLE_ROUTE_EGRES
S_INTERFACE(37)
```

While it fails there is a missing flow for ip address translation
```
(35): TABLE_ROUTE_EGRESS_LOOKUP
35-00        0       0 => SWITCH(0x0)               actions=drop
35-10        0       0 => ACTIVE_INTERFACE(0x16)   metadata=VALUE_PAIR(0x8000000a00000000/0xffffffff00000000)(0x0) actions=goto_table:TABLE_LOOKUP_IF_RL_TO_DP_RL(71)
35-20        0       0 => INTERFACE(0xa)[0x12]     metadata=VALUE_PAIR(0x8000000a00000000/0xffffffff00000000)(0x0) actions=write_metadata:0x70200000000000a,goto_table:TABLE_ROUTE_EGRESS_TRANSLATION(36)
35-20       19       0 => INTERFACE(0xb)[0x12]     metadata=VALUE_PAIR(0x8000000b00000000/0xffffffff00000000)(0x0) actions=write_metadata:0x70200000000000b,goto_table:TABLE_ROUTE_EGRESS_TRANSLATION(36)
```

This issue appears when a node starts sending the packets while all vna are still in its initialization phase. If you wait long enough until the initialisations of all vna are done, this issue won't happen.

**Comment by rakshasa** [comment]
Add logging to 'TranslationManager::load_static_addresses' and 'TranslationManager::added_static_address'.

That should give a clue as to what might be causing the above issues.

File: vnet/lib/vnet/core/translation_manager.rb [hunk 1]
```diff
@@ -80,8 +80,8 @@ def item_initialize(item_map)
     #
 
     def item_pre_install(item, item_map)
-      case item.class
```
Comment (rakshasa) [inline vnet/lib/vnet/core/translation_manager.rb]: This should be 'case item' and 'when T::SA'.

**Comment by rakshasa** [comment]
+1 when the above code is fixed.

**Pull request merged**